cmd/clusterctl/test/e2e/e2e_single_tenant_test.go

```diff
@@ -18,6 +18,65 @@ limitations under the License.
 
 package e2e
 
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	clusterctlconfig "sigs.k8s.io/cluster-api/cmd/clusterctl/pkg/client/config"
+)
+
+var _ = Describe("single tenant management cluster", func() {
+	var (
+		mgmt *ManagementCluster
+	)
+	BeforeEach(func() {
+		// Create a kind cluster configured for running the docker infrastructure provider and with all the images defined in the E2E test configuration.
+		var err error
+		mgmt, err = CreateManagementCluster(ctx, CreateManagementClusterInput{
+			// pass reference to the config for this E2E test that includes the name of the management cluster and the list of images to pre-load
+			config: config,
+		})
+		Expect(err).ToNot(HaveOccurred())
+		Expect(mgmt).ShouldNot(BeNil())
+	})
+
+	AfterEach(func() {
+		// Tears down the kind cluster
+		mgmt.Teardown(ctx, skipResourceCleanup)
```

Comment:
Don't we need Teardown() for the workload cluster here too?

Reply:
now it is addressed